test/health_cards/qbp_client_test.rb

```diff
@@ -0,0 +1,166 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'health_cards/qbp_client'
+
+class QBPClientTest < ActiveSupport::TestCase
+  setup do
+    @complete_patient = { patient_list: { id: 'J19X5',
+                                          assigning_authority: 'AIRA-TEST',
+                                          identifier_type_code: 'MR' },
+                          patient_name: { family_name: 'WeilAIRA',
+                                          given_name: 'BethesdaAIRA',
+                                          second_or_further_names: 'Delvene',
+                                          suffix: '' },
+                          mothers_maiden_name: { family_name: 'WeilAIRA',
+                                                 given_name: 'BethesdaAIRA',
+                                                 name_type_code: 'M' },
+                          patient_dob: '20170610',
+                          admin_sex: 'F',
+                          address: { street: '1113 Wollands Kroon Ave',
+                                     city: 'Hamburg',
+                                     state: 'MI',
+                                     zip: '48139',
+                                     address_type: 'P' },
+                          phone: { area_code: '810',
+                                   local_number: '2499010' } }
+
+    WebMock.allow_net_connect!
```

Comment:
Unit tests should not depend on external services. The requests should all be mocked.

Reply:
Stubbed all requests in recent commit using VCR library.